src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,48 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class encapsulates an event item
+ */
+public class Event extends Task {
+    protected LocalDate at;
```

Comment:
Maybe this variable could be named `timing` or `time` because `at` is a bit ambiguous and might mean location as well?